Sources/SwiftParserDiagnostics/ParseDiagnosticsGenerator.swift

```diff
@@ -470,6 +470,28 @@ public class ParseDiagnosticsGenerator: SyntaxAnyVisitor {
         message: { _ in .joinConditionsUsingComma },
         moveFixIt: { ReplaceTokensFixIt(replaceTokens: $0, replacement: trailingComma) }
       )
+    } else if let unexpected = node.unexpectedBetweenConditionAndTrailingComma,
+      let availability = node.condition.as(AvailabilityConditionSyntax.self),
+      let comparisonOperator = unexpected.oneTokenSatisfying(satisfying: { $0.tokenKind == .binaryOperator("==") }),
```

Comment:
I thought it might be convenient to create an ad-hoc function . Let me know if there's a better way to do it.